docs/4.3/faq.md

```diff
@@ -52,9 +52,9 @@ Scenario | Max Recommended Count | Proxy | Auth server
 Teleport nodes connected to auth server | 10,000 |2x  2-4 vCPUs/8GB RAM | 2x 4-8 vCPUs/16GB RAM
 Teleport nodes connected to proxy server (IoT) | 2,000* | 2x 2-4 vCPUs/8GB RAM |2x 4-8 vCPUs/16+GB RAM
 
-### Which version of Teleport is supported?
+## Which version of Teleport is supported?
 
-| Release       | LTS | Release Date         | Supported Until      | Minimum tsh version  |
+| Release       | Long Term Support | Release Date         | Supported Until      | Min tsh version  |
```

Comment:
Why did we change from `Minimum` to `Min`?

Reply:
I felt that we could shorten it to help these items display on screen better while not sacrificing a user's understanding of the title.

Reply:
I checked this on my phone, and we should make these tables scroll on mobile.
![image](https://user-images.githubusercontent.com/559288/87609363-35abfc80-c6b7-11ea-916c-ff0dfb1d26f8.png)